app/controllers/api_controller.rb

```diff
@@ -43,36 +44,44 @@ def area
       "Recent applications in the area (#{lat0},#{lng0}) (#{lat1},#{lng1})")
   end
 
+  def date_scraped
+    begin
+      date = Date.parse(params[:date_scraped])
+    rescue ArgumentError => e
+      raise e unless e.message == "invalid date"
+    end
+
+    if date
+      api_render(Application.where(date_scraped: date.beginning_of_day...date.end_of_day), "All applications collected on #{date}")
+    else
+      respond_to do |format|
+        format.js { render json: {error: "invalid date_scraped"}, status: 401 }
```

Comment:
Ahem. I think someone copied this line - the status code doesn't look right to me.